core/arch/arm/kernel/thread_a32.S

```diff
@@ -748,6 +748,56 @@ END_FUNC thread_unwind_user_mode
 FUNC thread_vect_table , :
 UNWIND(	.fnstart)
 UNWIND(	.cantunwind)
+#ifdef CFG_CORE_WORKAROUND_SPECTRE_BP
```

Comment:
I'm wondering if we shouldn't use a different config option here as we're protecting against a different attack and also the defense will be quite expensive in AArch64.

Reply:
Agreed. `CFG_CORE_WORKAROUND_SPECTRE_BP_SEC` maybe? (rationale: still Spectre branch prediction attack but entirely within secure world hence `_SEC`).

Reply:
Sounds good